src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,329 @@
+the web. What's more, playback can be done with either DASH or HLS.  This
+section will show you how to encrypt using clear key.
+
+The first thing you need to do is generate a key. You can use whatever method
+you want, but I'm going to use [OpenSSL](https://www.openssl.org/) to write
+sixteen random hex digits to a file. This step is necessary when preparing
+resources for both HLS and DASH.
+
+    openssl rand -out media.key 16
+
+Some versions of OpenSSL seem to create a file with white space and new line
+characters. Make sure these are absent or removed from `media.key` before
+proceeding.
+
+Use Shaka Packager to do the actual encryption. Use the content of the
+`media.key` file for both the `-key` and `-key-id` flags.
+
+Note: Technically, the key ID is supposed to be either the first 8 OR the
+first 16 hex digits of the key. Since packager requires the key to be 16
+digits and does not allow a 32 digit key, both flags use the same value.
+
+    packager \
+      input=myvideo.mp4,stream=audio,output=glocka.m4a \
+      input=myvideo.mp4,stream=video,output=glockv.mp4 \
+      --enable_fixed_key_encryption --enable_fixed_key_decryption \
+      -key INSERT_KEY_HERE -key_id INSERT_KEY_HERE
+
+This example implies that I've typed or pasted the key into the command line
+manually. There's no reason you can't get fancy.
+ 
+    packager \
+      input=myvideo.mp4,stream=audio,output=glocka.m4a \
+      input=myvideo.mp4,stream=video,output=glockv.mp4 \
+      --enable_fixed_key_encryption --enable_fixed_key_decryption \
+      -key "$(< media.key)" -key_id "$(< media.key)"
+
+### Widevine Encryption
+
+Unless your company completes the Master License Agreement with
+[Widevine]([http://www.widevine.com/contact.html](http://www.widevine.com/contact.html))
+, this type of encryption can really only be used for testing. Since this is
+covered without much explanation on the Shaka Packager READMEhere it goes.
+
+Everything in this command except the name of your files and the `--content_id`
+flag should be copied exactly from the example. The `--content_id` is 16 or 32
+random hex digits. Use the keys provided here instead of your own. (This is how
+Widevine works.)
+ 
+    packager \
+      input=glocken.mp4,stream=video,output=enc_video.mp4 \
+      input=glocken.mp4,stream=audio,output=enc_audio.m4a \
+      --enable_widevine_encryption \
+      --key_server_url "https://license.uat.widevine.com/cenc/getcontentkey/widevine_test" \
+      --content_id "16_Rand_Hex_Chrs" --signer "widevine_test" \
+      --aes_signing_key "1ae8ccd0e7985cc0b6203a55855a1034afc252980e970ca90e5202689f947ab9" \
+      --aes_signing_iv "d58ce954203b7c9a9a9d467f59839249"
+
+## Conclusion
+
+This does not cover everything you could do to a media file before posting it to
+the web, not by a longshot. To be fair, this subject is one deserving of a
+website that is itself as large as [developers.google.com/web/](/web/). I'm
```

Comment:
I'd take out this section; we're trying to create this sub-site for posting to the web here. Could reference developers.google.com/media/vp9 for VP9/WebM encoding if you'd like.